test/WebSites/BasicWebSite/Areas/Area1/Views/RemoteAttribute_Home/Create.cshtml

```diff
@@ -1,10 +1,7 @@
-﻿@model ValidationWebSite.Models.Person
+﻿@model BasicWebSite.Models.RemoteAttributeUser
 @{
-    object areaObject;
-    ViewContext.ActionDescriptor.RouteValueDefaults.TryGetValue("area", out areaObject);
-    var areaName = (areaObject as string) ?? "root";
-
-    ViewBag.Title = "Create in " + areaName + " area.";
+    Layout = "/Areas/Area1/Views/RemoteAttribute_Home/_Layout.cshtml";
+    ViewBag.Title = "Create in Area1 area.";
```

Comment:
Previously these tests were using attribute routing. Was this on purpose? I ask this because the route template that they were using is already part of the defined conventional routes and so felt unnecessary to have them defined and also they interfere with other routes(top most order) causing other tests to fail. When I removed the attribute routes, this way of getting area detail does not work `ViewContext.ActionDescriptor.RouteValueDefaults.TryGetValue("area", out areaObject);`.
Also aren't we covering the link generation of attribute routes somewhere else and so we can avoid it here as its not a pivot in this case.